### ISSUE TITLE
Option with Imperial unit

### ISSUE THREAD

**Issue by Matrix7399** [dylan-smith/DiveIntelligence#93]
Is it possible to add an option to show imperial units for the result?

**Comment by dylan-smith** [comment]
Absolutely, this is near the top of my list. I want to get gradient factors implemented first, then will probably tackle this next.
